### DISCUSSION TITLE
Build meta-leda image via kas tool - python3-grpcio_1.45.0.bb:do_compile failed

### DISCUSSION THREAD

**Discussion by MohamedAbdulslam** [eclipse-cdt/cdt#955] [General]
Hi experts
I'm new to Eclipse Leda, I am trying to build meta-leda via kas tool following Leda project documentation.
First, I've cloned the meta-leda repo (https://github.com/eclipse-leda/meta-leda)
Then I run the command 'kas build kas/poky-sdv-qemux86-64.yaml' to Initialize the build environment and start the build for QEMU
Unfortunately the build fails with this error related to python3-grpcio recipe:
| ERROR: 'python3 [setup.py](http://setup.py/) bdist_wheel ' execution failed.
| WARNING: exit code 1 from a shell command.
ERROR: Task (virtual:native:/data/test/meta-leda/build/../meta-openembedded/meta-python/recipes-devtools/python/python3-grpcio_[1.45.0.bb](http://1.45.0.bb/):do_compile) failed with exit code '1'
and I tried the following as a troubleshoot for this issue:
Checked all the dependencies required to build grpcio (protobuf , grpc-tools )
Checked host tools version like pip, setuptools and wheel
What are your recommendations to troubleshoot this error? is any missing step for me?

**Comment by jld01** [comment]
Hi @MohamedAbdulslam, this discussion area is for Eclipse CDT. I suggest you raise your question with the Eclipse Leda community: https://github.com/eclipse-leda/leda-distro/discussions
